app/components/dashboard/DcChart.vue

```
@@ -7629,6 +7558,14 @@ const setBgWindowZIndex = (id) => {
   }
 }
 
+const onFilteredSunburstChart = (i, {chart}) => {
```

ESLint

[error] A space is required after '{'.

(@stylistic/object-curly-spacing)